src/components/UserProfile/UserProfile.jsx

```diff
@@ -639,15 +699,12 @@ const UserProfile = props => {
               <TabPane tabId="4">
                 <ProjectsTab
                   userProjects={userProfile.projects || []}
-                  userTasks={tasks}
                   projectsData={props?.allProjects?.projects || []}
                   onAssignProject={onAssignProject}
                   onDeleteProject={onDeleteProject}
                   edit={hasPermission(requestorRole, 'editUserProfile', roles, userPermissions)}
                   role={requestorRole}
                   userPermissions={userPermissions}
-                  userId={props.match.params.userId}
-                  updateTask={onUpdateTask}
                 />
```

Comment:
I'm a little bit confused here. What's the purpose of removing `userTasks` and `updateTask` props here and also removing related functions? I can't see my task list in the projects tab when I test this branch.